tests/lean/run/dottedIdentNotation.lean

```diff
@@ -0,0 +1,212 @@
+import Lean
```

Comment:
Could you add some new interactive tests for the adjusted completion behavior, too?